AEPTarget.xcodeproj/project.pbxproj

```diff
@@ -822,7 +822,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/zsh;
-			shellScript = "cd ${PROJECT_DIR}\nif which swiftformat >/dev/null; then\n   swiftformat .\nelse\necho \"error: SwiftFormat not installed, download from https://github.com/nicklockwood/SwiftFormat\"\nfi\n";
+			shellScript = "cd ${PROJECT_DIR}\nif which ./Pods/SwiftLint/swiftlint >/dev/null; then\n  ./Pods/SwiftLint/swiftlint\nelse\n  echo \"error: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
i guess since `SwiftLint` is a dependency in the podfile now, maybe this error message should prompt the developer to run `pod update` instead of pointing to the github repo?

Reply:
I am of the same opinion, but I wanted to keep it consistent with the changes we made for Mobile Core https://github.com/adobe/aepsdk-core-ios/pull/722/files. I'll update it anyways.